sunscreen_compiler_common/src/context.rs

```diff
@@ -277,16 +378,13 @@ where
      */
     pub fn add_node(&mut self, operation: O) -> NodeIndex {
         #[cfg(feature = "debugger")]
-        {
-            let group_id = self.group_counter;
-            self.graph.add_node(NodeInfo {
-                operation,
-                group_id,
-            })
-        }
+        let group_id = self.group_counter;
```

Comment:
Add a TODO that this will query your trie structure thingy to get a real group id.